botcity/core/bot.py

```diff
@@ -1067,7 +1077,25 @@ def type_keys_with_interval(self, interval, keys):
             interval (int): Interval (ms) in which to press and release keys
             keys (list): List of keys to be pressed
         """
-        pyautogui.hotkey(*keys, interval=interval/1000.0)
+        formatted_keys = []
+        for key in keys:
+            if len(key) > 1:
+                key = key.lower()
+                key_value = keys_map.get(key, None)
+                if key_value:
+                    formatted_keys.append(key_value)
+                elif key in Key._member_names_:
+                    key_value = Key[key]
+                    formatted_keys.append(key_value)
+            else:
+                formatted_keys.append(key)
```

Comment:
What do you think of this approach in code?
```suggestion
            if len(key) <= 1:
                formatted_keys.append(key)
                continue

            key = key.lower()
            key_value = keys_map.get(key, None)

            if key_value is not None:
                formatted_keys.append(key_value)
            elif key in Key._member_names_:
                key_value = Key[key]
                formatted_keys.append(key_value)
```
That way, you isolate the if/else in just one if that isolates the situation itself.